packages/turf-line-offset/bench.js

```diff
@@ -0,0 +1,64 @@
+var along = require('./');
+var Benchmark = require('benchmark');
+var fs = require('fs');
+
+var line = {
+  type: "Feature",
+  properties: {},
+  geometry: {
+    type: "LineString",
+    coordinates: [
+      [
+        -77.0316696166992,
+        38.878605901789236
+      ],
+      [
+        -77.02960968017578,
+        38.88194668656296
+      ],
+      [
+        -77.02033996582031,
+        38.88408470638821
+      ],
+      [
+        -77.02566146850586,
+        38.885821800123196
+      ],
+      [
+        -77.02188491821289,
+        38.88956308852534
+      ],
+      [
+        -77.01982498168944,
+        38.89236892551996
+      ]
+    ]
+  }
+};
+
+var route = JSON.parse(fs.readFileSync(__dirname + '/test/fixtures/route.geojson'));
```

Comment:
Try to make the new `benchmark.js` files as minimalistic as possible, use a few of the latest modules as examples:
https://github.com/Turfjs/turf/blob/0cfbda88456a89c7cd988ac17afd6056a42f59a8/packages/turf-rhumb-distance/bench.js